src/server/pdc_server_region/pdc_server_data.c

```diff
@@ -5013,7 +5064,44 @@ PDC_Server_data_write_out(uint64_t obj_id, struct pdc_region_info *region_info,
     }
     if (is_contained == 0) {
         request_region->offset = lseek(region->fd, 0, SEEK_END);
-// printf("posix write for position %d with write size %u\n", 0, (unsigned)write_size);
+
+        // debug
+        /* fprintf(stderr, "==PDC_SERVER[%d]: %s posix write for position %d with write size %u\n", */
+        /*         pdc_server_rank_g, __func__, 0, (unsigned)write_size); */
+#ifdef ENABLE_ZFP
```

Comment:
This compression part is only relevant to the EQSIM code, or can it be used by any other test/code? Can we add something in the documentation for either case, including the option we need to enable/disable this?